dataland-frontend/tests/e2e/specs/prepopulation/AwaitPrepopulation.ts

```diff
@@ -37,6 +50,21 @@ describe("I want to ensure that the prepopulation has finished before executing
             nonFinancialResponse.matchingCompanies >= minimumNumberNonFinancialCompanies,
             `Found ${nonFinancialResponse.matchingCompanies} non-financial companies (Expecting at least ${minimumNumberNonFinancialCompanies})`
           );
+          const lksgResponse = await countCompanyAndDataIds(token, DataTypeEnum.Lksg);
```

Comment:
Is countCompanyAndDataIds named correctly? It seems to only count companies containing a data set of a certain type.

Reply:
it counts the number of companies that have a certain datatype and the number of datasets in total (which is not the same since on company could have multiple datasets) and hence I find the name fitting

Reply:
Yes, overread something